JSDemos/Demos/Charts/PieWithCustomLabels/description.md

```diff
@@ -1 +1,29 @@
-This demo illustrates how the PieChart component arranges labels in&nbsp;several columns. To&nbsp;maximize readability, the component ensures that labels do&nbsp;not overlap one another.
\ No newline at end of file
+This demo illustrates how to customize the PieChart component labels.
+
+The [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/).[label](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/label/) object allows you to specify the following settings:
+
+- [visible](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/label/#visible)    
+Specifies the visibility of point labels.
+
+- [position](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/label/#position)    
+Specifies a label position relative to the PieChart.
+
+    - `outside`   
+    The labels are outside the component.
+
+    - `columns`    
+    The labels are outside the component and are rearranged in columns. You can see the implementation in this demo.
+
+    - `inside`    
+    The labels are inside the points.
+
+- [customizeText](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/label/#customizeText)    
+A function that returns the text to be displayed by point labels.
+
+- [font](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/label/font/)    
+Specifies font properties for the label text. Includes multiple settings. This demo uses the [size](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/label/font/#size) option to change the font size.
```

Comment:
Well, if you already say "specifies font properties", it actually means that this property is actually a property set and, therefore, includes multiple settings. Therefore, we can safely remove "includes multiple settings".

```suggestion
Specifies font properties for label text. This demo uses the nested [size](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/label/font/#size) property to change the font size.
```